api/sample-code.md

```diff
@@ -28,14 +28,13 @@ $whmcsUrl = "https://www.yourdomain.com/path/to/whmcs/";
 // For WHMCS 7.2 and later, we recommend using an API Authentication Credential pair.
 // Learn more at http://docs.whmcs.com/API_Authentication_Credentials
 // Prior to WHMCS 7.2, an admin username and md5 hash of the admin password may be used.
-$username = "your_api_credential_identifier";
-$password = "your_api_credential_secret";
+$api_identifier = 'D4j1dKYE3g40VROOPCGyJ9zRwP0ADJIv';
```

Comment:
Thanks for the suggestion here.  Certainly happy to incorporate this. However could you replace the values in `$api_identifier` and `$api_secret` with placeholder values to make it clearer to other users these need to be populated with their own values?